samwise.js

```diff
@@ -1,26 +1,24 @@
 /**
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.